gapic-generator/lib/gapic/presenters/gem_presenter.rb

```diff
@@ -121,6 +121,34 @@ def iam_dependency?
         @api.files.map(&:name).any? { |f| f.start_with? "google/iam/v1/" }
       end
 
+      def library_documentation_url
+        gem_config(:library_documentation_url) || "https://googleapis.dev/ruby/#{name}/latest"
+      end
+
+      def product_documentation_url
+        gem_config :product_documentation_url
+      end
+
+      def api_id
+        gem_config :api_id
+      end
+
+      def free_tier?
+        gem_config(:free_tier) ? true : false
```

Comment:
seems like this and some other fields are used only in wrapper templates but are defined here

Reply:
This one will be used in the main generator. It's used in the README, and I want to do a follow-up for the main generator that includes a lot of this content there as well.